application/application.go

```diff
@@ -1,12 +1,17 @@
 package application
 
+import (
+	"github.com/almighty/almighty-core/account"
+)
+
 //An Application stands for a particular implementation of the business logic of our application
 type Application interface {
 	WorkItems() WorkItemRepository
 	WorkItemTypes() WorkItemTypeRepository
 	Trackers() TrackerRepository
 	TrackerQueries() TrackerQueryRepository
 	SearchItems() SearchRepository
+	Identities() account.IdentityRepository
```

Comment:
The individual modules should own their own interface. The reason they were moved up is simply because the Tracker module does it wrong and cause a cycle. We're missing 3 module to integrate between workitem and tracker. 

I see no reason to continue down the wrong path just because.